### PR TITLE
Install prerequisite packages for Longhorn on all nodes

### PR DESCRIPTION
**Pull request by divolgin**
Use tasks.sh script to install host dependencies for Longhorn after the daemon set is created

### CODE REVIEW

File: addons/longhorn/1.1.2/install.sh [hunk 1]
```diff
@@ -190,6 +138,48 @@ function validate_longhorn_ds() {
     fi
 }
 
+# if this is a multinode cluster, we need to prepare all hosts to run the daemonset
+function maybe_init_hosts() {
```
Comment (laverya) [inline addons/longhorn/1.1.2/install.sh:142]: I would assume that this function needs to be added to the template too

File: scripts/common/longhorn.sh [hunk 2]
```diff
@@ -0,0 +1,54 @@
+function longhorn_host_init() {
```
Comment (laverya) [inline scripts/common/longhorn.sh:1]: I'm not a huge fan of moving this to common instead of leaving it in the addon - why are we doing that?

**Review by laverya — DISMISSED** [review]

File: addons/longhorn/1.1.2/install.sh (hunk 1 above)
Comment (divolgin) [reply addons/longhorn/1.1.2/install.sh:142]: added

File: scripts/common/longhorn.sh (hunk 2 above)
Comment (divolgin) [reply scripts/common/longhorn.sh:1]: It is now called from tasks.sh

**Review by laverya — APPROVED** [review]

**Review by laverya — APPROVED** [review]

**Pull request merged**